gallery/src/pages/components/ha-alert.markdown

```diff
@@ -84,58 +84,43 @@ Actions must have a tab index of 0 so that they can be reached by keyboard-only
 
 The `title ` option should not be used without a description.
 
-<ha-alert alert-type="error" title="Error">
-	This is an error alert — <strong>check it out!</strong>
+<ha-alert alert-type="success" title="Success">
+  This is an success alert — check it out!
 </ha-alert>
 
-<ha-alert alert-type="warning">
-	<title>Warning</title>
-	This is an warning alert — <strong>check it out!</strong>
+```html
+<ha-alert alert-type="success" title="Success">
+  This is an success alert — check it out!
 </ha-alert>
+```
 
-<ha-alert alert-type="info">
-	<title>Info</title>
-	This is an info alert — <strong>check it out!</strong>
-</ha-alert>
+**Dismissable**
 
-<ha-alert alert-type="success">
-	<title>Success </title>
-	This is an success alert — <strong>check it out!</strong>
-</ha-alert>
+<ha-alert alert-type="success" dismissable>This is an success alert — check it out!</ha-alert>
 
 ```html
-<ha-alert alert-type="error" title="Error">
-	This is an error alert — <strong>check it out!</strong>
-</ha-alert>
-<ha-alert alert-type="warning">
-	<title>Warning</title>
-	This is an warning alert — <strong>check it out!</strong>
-</ha-alert>
-<ha-alert alert-type="info">
-	<title>Info</title>
-	This is an info alert — <strong>check it out!</strong>
-</ha-alert>
-<ha-alert alert-type="success">
-	<title>Success </title>
-	This is an success alert — <strong>check it out!</strong>
-</ha-alert>
+<ha-alert alert-type="success" dismissable>This is an success alert — check it out!</ha-alert>
 ```
 
-**Dismissable**
-
-*Documentation coming soon*
+**Slotted action**
 
-**Slotted icon**
+<ha-alert alert-type="success">This is an success alert — check it out!<mwc-button slot="action" label="Undo"></mwc-button></ha-alert>
 
-*Documentation coming soon*
+```html
+<ha-alert alert-type="success">This is an success alert — check it out!<mwc-button slot="action" label="Undo"></mwc-button></ha-alert>
```

Comment:
Add new lines to all examples in codeblocks to avoid causing horizontal scrollbars. Remember we're also looking at examples on mobile.

This is a screenshot on desktop:

![CleanShot 2022-01-14 at 15 31 03](https://user-images.githubusercontent.com/1444314/149598179-4cb86f36-cdb5-4a48-ab7e-031a05431c57.png)